src/GraphQL.Benchmarks/Benchmarks/DetailedBenchmark.cs

```diff
@@ -131,7 +132,7 @@ private void Run(BenchmarkInfo benchmarkInfo)
                     {
                         o.Schema = benchmarkInfo.Schema;
                         o.Query = benchmarkInfo.Query;
-                        o.Variables = benchmarkInfo.InputsString?.ToInputs();
+                        o.Variables = _serializer.Deserialize<Inputs>(benchmarkInfo.InputsString) ?? Inputs.Empty;
```

Comment:
Interesting change here - new API now returns `null` instead of `Inputs.Empty` previosly. I am sure that many will stumble about it. Dangerously.

Reply:
True, but `ExecutionOptions.Variables` does not require `Inputs.Empty` over `null`.  This code is incorrect and should not have ` ?? Inputs.Empty` added.

The stumbling will be in tests, if anywhere.

I suggest that after reviewing this PR, we add back the removed `ToInputs` extension method, with an obsolete attribute.  Perhaps the `FromJson` method also.  See migration document which clearly documents this change.

Reply:
> This code is incorrect and should not have ?? Inputs.Empty added.

Then why did you add `Inputs.Empty`?

Reply:
To recap: This code is incorrect and **I should not have added** `?? Inputs.Empty`.

Reply:
OK

Reply:
Fixed